client/sites-dashboard/components/sites-table-row-loading.tsx

```diff
@@ -0,0 +1,69 @@
+import styled from '@emotion/styled';
+
+const Row = styled.tr`
+	line-height: 2em;
+	border-bottom: 1px solid #eee;
+`;
+
+const Column = styled.td< { mobileHidden?: boolean } >`
+	padding-top: 12px;
+	padding-bottom: 12px;
+	padding-right: 24px;
+	vertical-align: top;
+`;
+
+const Loading = styled.div`
+	animation: pulse-light 1.8s ease-in-out infinite;
+	background-color: var( --color-neutral-10 );
```

Comment:
It's too bad we can't reuse the `placeholder()` SCSS mixin here in Emotion, so that we resuse the same animation as the rest of Calypso.
https://github.com/Automattic/wp-calypso/blob/7e0a33b4ac44dae1aca27d2d403bbb9a2a476431/client/assets/stylesheets/shared/mixins/_placeholder.scss#L6

We could add a SCSS file to the project just for this one thing. Up to you, it's probably fine to have our own version (I bet there's already a lot of duplicates of this animation in Calypso).

Reply:
I've created a new component, `LoadingPlaceholder` inside packages. Now all the logic and styles live in that component with new enhancements such as adding an animation delay.
I've also added new examples in the storybook.

<img width="1013" alt="Screenshot 2022-08-05 at 17 47 36" src="https://user-images.githubusercontent.com/779993/183123671-1af317e6-7401-43f8-8890-ac10675c3c41.png">

With this approach, our components will be more reusable in different applications.

https://github.com/Automattic/wp-calypso/blob/172c2ed745b7ad9c4a986154494c0104c8956fdb/packages/components/src/loading-placeholder/loading-placeholder.tsx#L1-L21

Reply:
I've created a new component, `LoadingPlaceholder` inside packages. Now all the logic and styles live in that component with new enhancements such as adding an animation delay.
I've also added new examples in the storybook.

<img width="1013" alt="Screenshot 2022-08-05 at 17 47 36" src="https://user-images.githubusercontent.com/779993/183123671-1af317e6-7401-43f8-8890-ac10675c3c41.png">

With this approach, our components will be more reusable in different applications.

https://github.com/Automattic/wp-calypso/blob/172c2ed745b7ad9c4a986154494c0104c8956fdb/packages/components/src/loading-placeholder/loading-placeholder.tsx#L1-L21